content/blog/2020-02-17-a-public-reddit-dataset.md

```diff
@@ -317,10 +317,10 @@ refine these existing methods. And there’s almost certainly room to push the
 state of the art in asshole detection!
 
 If you're interested in learning more about using Reddit data, check out
-[pushshift.io](https://pushshift.io/), a database that contains basically all of
-Reddit's content (so why make this dataset? I wanted to remove some of the
-barriers to analyzing text from r/AmItheAsshole by providing an
-already-processed and cleaned version of the data that can be downloaded with a
-line of code; pushshift takes some work). You might use pushshift's API and/or
-praw to augment this dataset in some way- perhaps to compare activity in this
-subreddit with another, or broader patterns on Reddit.
+[pushshift.io](https://pushshift.io/), a database that contains all of Reddit's
```

Comment:
This one is also trying to say something important. I still don't see many of these words as condescending but maybe I'm still wrong. I would agree to replace "basically" here with something more specific (as explained in https://github.com/iterative/dvc.org/pull/1396#discussion_r437836708).

Reply:
Actually, let's roll this one back and not touch blog posts to make this easier. Blog posts have specific authors and the tone in their posts is up to them 🙂